Cargo.toml

```diff
@@ -24,6 +24,7 @@ rustc-serialize = "^0.3.15"
 num = "^0.1.25"
 lazy_static = "^0.1.14"
 quick-error = "^0.1.3"
+pulldown-cmark={ git="https://github.com/google/pulldown-cmark.git"}
```

Comment:
Is there a reason you don't use the [version from Crates.io](https://crates.io/crates/pulldown-cmark)?